cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -310,6 +310,7 @@ func translateStubDomainOfKubeDNSToProxyCoreDNS(dataField string, kubeDNSConfigM
 			pStanza["body"] = [][]string{
 				{"errors"},
 				{"cache", "30"},
+				{"loop"},
```

Comment:
Why no loadbalance here?

Reply:
That bit is for creating stub domain stanzas, so coredns is just forwarding here.  I was thinking we would not want to randomize answers from stubdomains, since those servers may already be deliberately ordering the answers.